platforms/hyperledger-besu/charts/besu-connector/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: besu_connector
-description: A Helm chart for Cactus Besu Connector
+description: "Hyperledger Besu: A Helm chart for deploy Cactus Besu Connector"
```

Comment:
Please change the description to: "Hyperledger Besu: Deploys Cactus Besu Connector" etc. "A Helm chart for .." can be removed as we know it is a helm chart already.

Reply:
Done